nevergrad/optimization/base.py

```diff
@@ -518,6 +518,10 @@ def recommend(self) -> p.Parameter:
             The candidate with minimal loss. :code:`p.Parameters` have field :code:`args` and :code:`kwargs` which can be directly used
             on the function (:code:`objective_function(*candidate.args, **candidate.kwargs)`).
         """
+        if self.num_objectives > 1:
+            raise RuntimeError(
+                "No best candidate in MOO. Use pareto_front function instead to get the set of all non-dominated candidates."
```

Comment:
Probably even more explicit:
```suggestion
                "No best candidate in MOO. Use optimizer.pareto_front() instead to get the set of all non-dominated candidates."
```